eland/query.py

```diff
@@ -215,10 +215,13 @@ def composite_agg_start(
             sources.append({bucket_agg_name: bucket_agg})
         self._composite_aggs.clear()
 
-        aggs = {
-            "composite": {"size": size, "sources": sources},
-            "aggregations": self._aggs.copy(),
+        aggs: Dict[str, Dict[str, Any]] = {
+            "composite": {"size": size, "sources": sources}
         }
+
+        if self._aggs:
```

Comment:
Nice, no guarantee there'll actually be `aggs` if we're simply calling `count()`